### ISSUE TITLE
[Accessibility]: `N/A` hard to understand when read aloud

### ISSUE THREAD

**Issue by efritz** [sourcegraph/sourcegraph#35189]
### Audit type

Screen reader navigation

### User journey audit issue

#33516, #33518

### Problem description

<img width="884" alt="n:a" src="https://user-images.githubusercontent.com/103087/167504500-6c228e31-688b-4262-ba26-14d18129ffd1.png">

I couldn't understand the text for `N/A` here.

### Expected behavior

Rename text or add an aria-label that can say `not applicable` or some other long-form version that can be understood.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
Reopening, I'll fix this one, just a quick copy change

**Comment by sourcegraph-bot-2** [comment]
Heads up @taylorsperry @jasongornall - the "team/frontend-platform" label was applied to this issue.
